code/obj/item/clothing/hats.dm

```diff
@@ -324,7 +324,8 @@ proc/filter_trait_hats(var/type)
 									"camera" = /obj/item/camera,
 									"audiolog" = /obj/item/device/audio_log ,
 									"flashlight" = /obj/item/device/light/flashlight,
-									"glasses" = /obj/item/clothing/glasses)
+									"glasses" = /obj/item/clothing/glasses,
+									"goggles" = /obj/item/clothing/glasses) //For the lizard mutatrance that say glassssssses
```

Comment:
Have you verified this works? From what I can read, it looks like the examine description will be fucked up

Reply:
I did test it, but entirely forgot that you can examine the hat, and yeah, the examine description does not work.
Do you think it would be better to replace "glasses" by "goggles" entirely, or perhaps some other word like "eyewear". 
Or just scrap the thing entirely?